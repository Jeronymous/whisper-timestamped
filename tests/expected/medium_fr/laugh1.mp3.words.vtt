WEBVTT

00:00.020 --> 00:01.600
Rires

