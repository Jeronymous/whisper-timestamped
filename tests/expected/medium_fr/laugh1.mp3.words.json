{
  "text": " Rires",
  "segments": [
    {
      "id": 0,
      "seek": 0,
      "start": 0.02,
      "end": 1.6,
      "text": " Rires",
      "tokens": [
        50364,
        497,
        3145,
        50576
      ],
      "temperature": 0.0,
      "avg_logprob": -1.0973867416381835,
      "compression_ratio": 0.38461538461538464,
      "no_speech_prob": 0.5563598275184631,
      "words": [
        {
          "text": "Rires",
          "start": 0.02,
          "end": 1.6
        }
      ]
    }
  ],
  "language": "fr"
}